### PR TITLE
HBASE-27762 Include EventType and ProcedureV2 pid in logging via MDC

### PR DESCRIPTION
**Pull request by ndimiduk**

### CODE REVIEW

**Comment by ndimiduk** [comment]
With this change, we get log messages like,

```
2023-03-27T15:46:01,930 INFO  [RS_CLAIM_REPLICATION_QUEUE-regionserver/scipio:0-0 ({event_type=RS_CLAIM_REPLICATION_QUEUE, pid=33})] replication.ZKReplicationQueueStorage(391): Atomically moving scipio,51464,1679924724756/region_replica_replication's WALs to scipio,51466,1679924724781
```

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 18s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 33s |  master passed  |
| +1 :green_heart: |  compile  |   1m 52s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 27s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 25s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 19s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 52s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 52s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 26s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 40s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 290m 38s |  root in the patch passed.  |
|  |   | 323m  9s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | unit javac javadoc shadedjars compile |
| uname | Linux c6077fe237eb 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / a6dad700db |
| Default Java | Eclipse Adoptium-11.0.17+8 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/1/testReport/ |
| Max. process+thread count | 4491 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/1/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 57s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 21s |  master passed  |
| +1 :green_heart: |  compile  |   1m 50s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 58s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 19s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 16s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 44s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 44s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 49s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 21s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 323m 49s |  root in the patch failed.  |
|  |   | 355m 17s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | unit javac javadoc shadedjars compile |
| uname | Linux 6f9a4f8e5eb9 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / a6dad700db |
| Default Java | Temurin-1.8.0_352-b08 |
| unit | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/1/artifact/yetus-jdk8-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/1/testReport/ |
| Max. process+thread count | 2706 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/1/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 19s |  master passed  |
| +1 :green_heart: |  compile  |   5m 54s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 20s |  master passed  |
| +1 :green_heart: |  spotless  |   1m  9s |  branch has no errors when running spotless:check.  |
| -1 :x: |  spotbugs  |  10m 11s |  root in master has 4 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m  0s |  the patch passed  |
| +1 :green_heart: |  compile  |   8m 23s |  the patch passed  |
| +1 :green_heart: |  javac  |   8m 23s |  the patch passed  |
| -0 :warning: |  checkstyle  |   1m 32s |  root: The patch generated 2 new + 0 unchanged - 0 fixed = 2 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  18m 54s |  Patch does not cause any errors with Hadoop 3.2.4 3.3.4.  |
| -1 :x: |  spotless  |   0m 51s |  patch has 36 errors when running spotless:check, run spotless:apply to fix.  |
| +1 :green_heart: |  spotbugs  |  14m 35s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 59s |  The patch does not generate ASF License warnings.  |
|  |   |  89m  6s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | dupname asflicense spotless javac spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux f17a5291fc79 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / efb360d33f |
| Default Java | Eclipse Adoptium-11.0.17+8 |
| spotbugs | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/artifact/yetus-general-check/output/branch-spotbugs-root-warnings.html |
| checkstyle | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/artifact/yetus-general-check/output/diff-checkstyle-root.txt |
| spotless | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/artifact/yetus-general-check/output/patch-spotless.txt |
| Max. process+thread count | 176 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/console |
| versions | git=2.34.1 maven=3.8.6 spotbugs=4.7.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-server/src/main/java/org/apache/hadoop/hbase/executor/EventHandler.java [hunk 1]
```diff
@@ -96,6 +97,10 @@ public EventHandler prepare() throws Exception {
   public void run() {
     Span span = TraceUtil.getGlobalTracer().spanBuilder(getClass().getSimpleName())
       .setParent(Context.current().with(parent)).startSpan();
+    // assume that this is the top of an execution on a new or reused thread, that we're safe to
+    // blast any existing MDC state.
+    MDC.clear();
```
Comment (Apache9) [inline hbase-server/src/main/java/org/apache/hadoop/hbase/executor/EventHandler.java:102]: We only call clear at the beginning? Is it possible to clear MDC at the end of a execution?

Comment (ndimiduk) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/executor/EventHandler.java:102]: Sure, seems no harm in that. You'd like to see it clear on both beginning and end?

Comment (Apache9) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/executor/EventHandler.java:102]: I prefer we do something like
```
MDC.put("blabla", "blabla");
try {
  ...
} finally {
  MDC.clear();
}
```

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 18s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  2s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 23s |  master passed  |
| +1 :green_heart: |  compile  |   1m 52s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 26s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 24s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 18s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 50s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 50s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 21s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 24s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 268m 10s |  root in the patch failed.  |
|  |   | 298m 50s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | unit javac javadoc shadedjars compile |
| uname | Linux 9eabb38d82cb 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / efb360d33f |
| Default Java | Eclipse Adoptium-11.0.17+8 |
| unit | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/artifact/yetus-jdk11-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/testReport/ |
| Max. process+thread count | 4796 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 39s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 26s |  master passed  |
| +1 :green_heart: |  compile  |   2m 14s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 25s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 47s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 18s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 15s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 27s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 27s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m  8s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 50s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 364m 39s |  root in the patch passed.  |
|  |   | 403m 51s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | unit javac javadoc shadedjars compile |
| uname | Linux 887903d1989b 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / efb360d33f |
| Default Java | Temurin-1.8.0_352-b08 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/testReport/ |
| Max. process+thread count | 4394 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/2/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  3s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 25s |  master passed  |
| +1 :green_heart: |  compile  |   4m 39s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m  8s |  master passed  |
| +1 :green_heart: |  spotless  |   0m 42s |  branch has no errors when running spotless:check.  |
| -1 :x: |  spotbugs  |   7m  6s |  root in master has 4 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 17s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 45s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 45s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m  6s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  12m 34s |  Patch does not cause any errors with Hadoop 3.2.4 3.3.4.  |
| +1 :green_heart: |  spotless  |   0m 41s |  patch has no errors when running spotless:check.  |
| +1 :green_heart: |  spotbugs  |   9m  8s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 31s |  The patch does not generate ASF License warnings.  |
|  |   |  60m  5s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | dupname asflicense spotless javac spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux f9ea9b4f86ba 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / e5620e26a2 |
| Default Java | Eclipse Adoptium-11.0.17+8 |
| spotbugs | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/artifact/yetus-general-check/output/branch-spotbugs-root-warnings.html |
| Max. process+thread count | 177 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/console |
| versions | git=2.34.1 maven=3.8.6 spotbugs=4.7.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 19s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 15s |  master passed  |
| +1 :green_heart: |  compile  |   1m 53s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 22s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 26s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 21s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 51s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 51s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 21s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 25s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 270m 38s |  root in the patch passed.  |
|  |   | 301m 21s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | unit javac javadoc shadedjars compile |
| uname | Linux a90d70587342 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / e5620e26a2 |
| Default Java | Eclipse Adoptium-11.0.17+8 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/testReport/ |
| Max. process+thread count | 4618 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 24s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m  1s |  master passed  |
| +1 :green_heart: |  compile  |   1m 31s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 34s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 45s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 41s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 27s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 27s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 29s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 43s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 323m  0s |  root in the patch failed.  |
|  |   | 349m 57s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5145 |
| Optional Tests | unit javac javadoc shadedjars compile |
| uname | Linux ef16b9b96e30 5.4.0-1094-aws #102~18.04.1-Ubuntu SMP Tue Jan 10 21:07:03 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / e5620e26a2 |
| Default Java | Temurin-1.8.0_352-b08 |
| unit | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/artifact/yetus-jdk8-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/testReport/ |
| Max. process+thread count | 2244 (vs. ulimit of 30000) |
| modules | C: hbase-logging hbase-server . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5145/3/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by ndimiduk** [comment]
Test failure is https://issues.apache.org/jira/browse/HBASE-27720 .

**Review by Apache9 — APPROVED** [review]

**Pull request merged**